js/src/chrome/projects/chrome.projects.js

```diff
@@ -624,6 +624,28 @@ Wu.Chrome.Projects = Wu.Chrome.extend({
 		var allUsers = _.sortBy(_.toArray(app.Users), function (u) {
 			return u.store.firstName;
 		});
+		var checkedUsers = {};
+		var itemsContainers = [];
+		
+		function onKeyUp(e) {
```

Comment:
better refactored to `this` namespace, instead of function inside funciton... seems there might be an issue with `this._access`, so refactoring is a good place to start. 

Same issues here as in https://github.com/systemapic/systemapic.js/pull/46/files